docs/man/dune.1.md

```diff
@@ -0,0 +1,285 @@
+    with maximum amount of MAX_VALUE and initial value of INITIAL_VALUE.
+
+**`--send-action <ACCOUNT> <ACTION> <DATA> <PERMISSION>`**
+
+    Send an action to a specified account with given data and permission.
+
+**`--get-table <ACCOUNT> <SCOPE> <TABLE>`**
+
+    Print data from a given table.
+
+**`--activate-feature <CODENAME>`**
+
+    Activate a given protocol feature.
+
+**`--list-features`**
+
+    Print a list of available protocol features.
+
+**`--version`**
+
+    Display the current version of DUNE.
+
+**`--version-all`**
+
+    Display the current versions of DUNE, CDT, and leap.
+
+**`--debug`**
+
+    Print additional information useful for debugging, such as running docker commands.
+
+**`--upgrade`**
+
+    Upgrade DUNE image to the latest version.
+
+**`--leap [LEAP_VERSION]`**
+
+    Set the version of leap. If no version is provided, display available leap versions.
+
+**`--cdt [CDT_VERSION]`**
+
+    Set the version of CDT (Contract Development Toolkit). If no version is provided, display available CDT versions.
+
+**`--create-project <PROJ_NAME> <DIR>`**
+
+    Create a smart contract project at the specified location.
+
+**`--add-app <PROJ_DIR> <APP_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+
+    Add an application to the specified smart contract project.
+
+**`--add-lib <PROJ_DIR> <LIB_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+
+    Add a library to the specified smart contract project.
+
+**`--add-dep <PROJ_DIR> <OBJ_NAME> <DEP_NAME> [LOCATION] [TAG/RELEASE] [HASH]`**
+
+    Add a dependency to the specified smart contract project.
+
+**`--remove-app <PROJ_DIR> <APP_NAME>`**
+
+    Remove an application from the specified smart contract project.
+
+**`--remove-lib <PROJ_DIR> <LIB_NAME>`**
+
+    Remove a library from the specified smart contract project.
+
+**`--remove-dep <PROJ_DIR> <OBJ_NAME> <DEP_NAME>`**
+
+    Remove a dependency from the specified smart contract project.
+
+**`--update-app <PROJ_DIR> <APP_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+
+    Update an application in the given smart contract project.
+
+**`--update-lib <PROJ_DIR> <LIB_NAME> <LANG> [CMPLR_OPTS] [LINK_OPTS]`**
+ 
+    Update a library in the given smart contract project.
+
+**`--update-dep <PROJ_DIR> <OBJ_NAME> <DEP_NAME> [LOCATION] [TAG/RELEASE] [HASH]`**
+
+    Update a dependency in the given smart contract project.
+ 
+**`--build-project <PROJ_DIR>`**
+
+    Build the given smart contract project.
+                        
+**`--clean-build-project <PROJ_DIR>`**
+
+    Clean the given project and rebuild it from scratch.
+                        
+**`--validate <PROJ_DIR>`**
+
+    Validate the given smart contract project.
+                        
+**`--populate <PROJ_DIR>`**
+
+    Populate the given smart contract project.
+
+# BUGS
+
+Please submit bug reports online at https://github.com/AntelopeIO/DUNE/issues
+
+# SEE ALSO
+
+For more details consult the full documentation and sources https://github.com/AntelopeIO/DUNE
```

Comment:
We might want to add more text here? IDK.  Something like:

```
This page is part of version 1.1.0 of Docker Utilities for Node Execution. A description of the project, information 
about reporting bugs, and the latest version of this page can be found at https://github.com/AntelopeIO/DUNE/.
```
Again, @iamveritas 's input is appreciated.

Reply:
the version is already listed on the top. to me it is enough as it is.

Reply:
the ALL caps titles are a burden for my eyes though :)
is this style necessary for the man standard?

Reply:
@iamveritas It would look bad if we were out of sync with the standard, I believe. To be honest, this is simply a header, though, and most users probably won't notice it. They will look at the body of the page.

Reply:
@iamveritas We agreed with @ScottBailey  to remove all information about the version at all, because, as I understand, the version is generating automatically and we risk to break version consistency.

Here is a manual how to write man pages: https://man7.org/linux/man-pages/man7/man-pages.7.html


Requirement for the title from this document:

       title  The title of the man page, written in all caps (e.g., MAN-
              PAGES).